CONTRIBUTING.md

```diff
@@ -0,0 +1,33 @@
+# Contributing
```

Comment:
can you link this in the main `README.md` as well

Reply:
will do